.tekton/integration-service-pull-request.yaml

```diff
@@ -160,18 +160,83 @@ spec:
       workspaces:
       - name: basic-auth
         workspace: git-auth
+    - name: sealights-go-instrumentation
+      runAfter:
+        - clone-repository
+      taskRef:
+        params:
+        - name: name
+          value: sealights-go-oci-ta
+        - name: bundle
+          value: quay.io/konflux-ci/tekton-catalog/task-sealights-go-oci-ta:0.1@sha256:3ee87d28065205ee5f1e32ec012cac4b167af6525f7504a9657612e4387d0eae
+        - name: kind
+          value: task
+        resolver: bundles
+      params:
+        - name: SOURCE_ARTIFACT
+          value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+        - name: go-version
+          value: "1.22"
```

Comment:
note: we'll need to document that this value needs to be updated when we change go versions.
@flacatus would this task report that the go versions do not match, or is it something that we could only catch manually?